forge.env.d.ts

```diff
@@ -15,6 +17,9 @@ declare global {
     }
   }
 
+  // eslint-disable-next-line no-var
+  var k6StudioState: k6StudioState
```

Comment:
We have many variables that are mutated across different handlers. This `k6StudioState` is meant to replace them since it's globally accessible.